cmd/otelcol/config/collector/otlp_config_linux.yaml

```diff
@@ -34,32 +34,29 @@ processors:
   batch:
   # Enabling the memory_limiter is strongly recommended for every pipeline.
   # Configuration is based on the amount of memory allocated to the collector.
-  # The configuration below assumes 2GB of memory. In general, the ballast
-  # should be set to 1/3 of the collector's memory, the limit should be 90% of
-  # the collector's memory up to 2GB, and the spike should be 25% of the
-  # collector's memory up to 2GB. The simplest way to specify the ballast size is
-  # set the value of SPLUNK_BALLAST_SIZE_MIB env variable. This will overrides
+  # In general, the ballast should be set to 1/3 of the collector's memory, the limit
+  # should be 90% of the collector's memory up to 2GB. The simplest way to specify the
+  # ballast size is set the value of SPLUNK_BALLAST_SIZE_MIB env variable. This will override
   # the value of --mem-ballast-size-mib command line flag. If SPLUNK_BALLAST_SIZE_MIB
   # is not defined then --mem-ballast-size-mib command line flag must be manually specified.
   # For more information about memory limiter, see
   # https://github.com/open-telemetry/opentelemetry-collector/blob/master/processor/memorylimiter/README.md
   memory_limiter:
     ballast_size_mib: ${SPLUNK_BALLAST_SIZE_MIB}
     check_interval: 2s
-    limit_percentage: ${SPLUNK_MEMORY_LIMIT_PERCENTAGE}
-    spike_limit_percentage: ${SPLUNK_MEMORY_SPIKE_PERCENTAGE}
+    limit_mib: ${SPLUNK_MEMORY_LIMIT_MIB}
 
 exporters:
   # Traces
-  otlp:
+  otlphttp:
     endpoint: "https://ingest.${SPLUNK_REALM}.signalfx.com/v2/trace/otlp"
```

Comment:
should this be `traces_endpoint` to avoid it becoming `v2/trace/otlp/v1/traces`?  https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlphttpexporter#otlphttp-exporter

Reply:
nice catch -- yes!